src/library/scala/StringContext.scala

```diff
@@ -92,7 +93,12 @@ case class StringContext(parts: String*) {
    *          if a `parts` string contains a backslash (`\`) character
    *          that does not start a valid escape sequence.
    */
-  def s(args: Any*): String = standardInterpolator(treatEscapes, args)
+  // The implementation is hardwired to `scala.tools.reflect.MacroImplementations.macro_StringInterpolation_s`
+  // Using the mechanism implemented in `scala.tools.reflect.FastTrack`
+  // and is logically the same as [[standardInterpolator(treatEscapes, args)]]
+  def sNew[A >: Any](args: A*): String = macro ???
+  def s[A >: Any](args: A*): String = standardInterpolator(treatEscapes, args)
+  private def sInternal(args: Any*): String = standardInterpolator(treatEscapes, args)
```

Comment:
I am not sure the mechanism for the bootstrap, so tried to re-code uses of s in StringContext to use the exising implementation, so when the target and the bootstrap is agreed, this may be removed if it doesnt introduce a circular dependency